src/main/java/qna/domain/Question.java

```diff
@@ -126,8 +127,32 @@ public void setAnswers(List<Answer> answers) {
     }
 
     public Answer getLastAnswer() {
-        return answers.size() > 0? answers.get(answers.size() - 1) : null;
+        if (answers.size() > 0)
+            return answers.get(answers.size() - 1);
```

Comment:
```suggestion
        if (answers.size() > 0) {
            return answers.get(answers.size() - 1);
        }
```

자바에서는 한 라인도 괄호로 감싸주는 것이 관례입니다 :)